tests/plugins/tableresize/manual/maximizeddivarea.md

```diff
@@ -0,0 +1,15 @@
+@bender-ui: collapsed
+@bender-tags: bug, 909, 1959, 4.14.1
+@bender-ckeditor-plugins: toolbar, wysiwygarea, sourcearea, divarea, tableresize, maximize
+
+1. Maximize editor by pressing maximize button.
+
+1. Try to resize table inside editor.
+
+### Expected
+
+Table is able to be resized.
```

Comment:
```suggestion
It is possible to resize the table or any of its columns (using CKEditor 4 resizer).
```

Reply:
The important thing here is that older IEs has also it's native resizer which is a different thing:

![](https://user-images.githubusercontent.com/1061942/83138033-7db49700-a0ea-11ea-8ab6-67706cf27da4.png)

so it should be mentioned explicitly that it is about CKEditor 4 resizer.